test/e2e/node/security_context.go

```diff
@@ -74,10 +74,25 @@ var _ = SIGDescribe("Security Context [Feature:SecurityContext]", func() {
 		pod := scTestPod(false, false)
 		userID := int64(1001)
 		pod.Spec.SecurityContext.RunAsUser = &userID
-		pod.Spec.Containers[0].Command = []string{"sh", "-c", "id -u"}
+		pod.Spec.Containers[0].Command = []string{"sh", "-c", "id"}
 
 		f.TestContainerOutput("pod.Spec.SecurityContext.RunAsUser", pod, 0, []string{
-			fmt.Sprintf("%v", userID),
+			fmt.Sprintf("uid=%v", userID),
+			"gid=0",
+		})
+	})
+
+	It("should support pod.Spec.SecurityContext.RunAsUser And pod.Spec.SecurityContext.RunAsGroup", func() {
```

Comment:
it might be better to put the `[Feature:RunAsGroup]` on the new testcases in particular